### DISCUSSION TITLE
User Profile Photo | Filament Admin Panel

### DISCUSSION THREAD

**Discussion by MarJose123** [filamentphp/filament#3529] [Help] [answered]
Are there already a Image upload helper or plugin similar to this but in form not in table column.
```
ImageColumn::make('author.avatar')->rounded()
```
I want to achieve similar to Laravel jetstream Profile management it has a capability to upload a user photo.
<img width="1274" alt="profile-management d7eec7d1" src="https://user-images.githubusercontent.com/18107626/185027508-7cc55028-b55a-4abe-a984-448c8451c4d9.png">

**Comment by mikrosmile** [comment, marked as answer]
`FileUpload::make('picture')
                    ->avatar()`

**Reply by MarJose123** [reply]
> `FileUpload::make('picture') ->avatar()`

I'm receiving `NOT FOUND` and this is the link on my browser console error 
```
GET http://localhost/storage/profile/NNpRkpc46MBpjvsIauEUr7Ij28SFbk-metacG5nZWdnLnBuZw==-.png
```
I'm done doing the `php artisan storage:link` but still no luck.

```php
  Forms\Components\FileUpload::make('picture')
                        ->avatar()
                        ->disk('public')
                        ->directory('profile')
                        ->image()
                        ->visibility('public')
```

here my `filesystem.php`
```php
      'disks' => [

        'local' => [
            'driver' => 'local',
            'root' => storage_path('app'),
            'throw' => false,
        ],

        'public' => [
            'driver' => 'local',
            'root' => storage_path('app/public'),
            'url' => env('APP_URL').'/storage',
            'visibility' => 'public',
            'throw' => false,
        ],

```

**Reply by danharrin** [reply]
Check your APP_URL

**Reply by MarJose123** [reply]
its localhost

**Reply by MarJose123** [reply]
I found the issue, it because of `php artisan serve` and the `APP_URL` indicate only the `http://localhost` not including the port it was using.

I've updated it to `http://localhost:8000`
